rhv_checks.py

```diff
@@ -338,6 +340,54 @@ def check_vms_distributed_hosts(system, warn=5, crit=10):
         sys.exit(3)
 
 
+def is_service_active(ssh, name):
```

Comment:
Since this is a utility really, can we mark it as private with name like `_is_service_active` ?  I'd also like to see if we can move it to a different file/module where we can add any other utilities in the future.

Reply:
@kedark3 I was thinking to move it to a different module... Though I don't understand marking it with `_`. I think there's a potential for these functions to be used in different checks